recipe/meta.yaml

```
@@ -31,6 +33,7 @@ build:
 
 requirements:
   build:
+    - {{ stdlib("c") }}
```

We are still working through adding `{{ stdlib("c") }}`

For now would just add `sysroot` and pin it to `2.17`. Would re-render after to propagate changes through the feedstock

```suggestion
    - sysroot_{{ target_platform }} 2.17  # [linux]
```